sql/report_queries/pick_list/pick_list.sql

```diff
@@ -97,11 +90,11 @@ instance_format_names AS (
 ),
 identifiers_extract AS (
     SELECT
-        inst.id AS instance_id,
-        JSON_EXTRACT_PATH_TEXT(JSON_ARRAY_ELEMENTS(JSON_EXTRACT_PATH(inst.data, 'identifiers')),'identifierTypeId') :: VARCHAR AS type_id,
-        JSON_EXTRACT_PATH_TEXT(JSON_ARRAY_ELEMENTS(JSON_EXTRACT_PATH(inst.data, 'identifiers')),'value') :: VARCHAR AS value
+        inst.instance_id AS instance_id,
+        inst.identifier_type_id AS type_id,
+        inst.identifier_type_name AS value
```

Comment:
I assume `inst.identifier_type_name` is supposed to be `inst.identifier`

Reply:
Thanks for catching that Axel. I just made that change in the file.